test/bib-serializations-test.js

```diff
@@ -453,6 +453,58 @@ describe('Bib Serializations', function () {
       })
     })
 
+    it('should extract other parallel fields', () => {
+      return Bib.byId('bParallels').then((bib) => {
+        return ResourceSerializer.serialize(bib).then((serialized) => {
+          const expectedParallelDisplayField = [
+            { fieldName: 'publicationStatement',
+               index: undefined,
```

Comment:
I think if you update to `https://github.com/NYPL-discovery/discovery-store-models/tree/v1.4.1` you should be able to add "index" properties to the `bParallels` fixture and have them come through.